### PR TITLE
[Manage Groups] Highlight the selected apps in dark mode

### LINKED ISSUE #4382: [Manage Groups] Highlight the selected apps in dark mode
### Is your proposal related to a problem?

The selection is not clearly visible when the apps are selected for a group (in dark mode)

### Describe the solution you'd like

Change the color for the selected apps to be visible

### Steps to reproduce the issue
- Go to the `Manage Groups` from the dashboard
- Toggle dark mode
- Open a group
- Select apps to add to the group

https://user-images.githubusercontent.com/25361949/195290208-93a463f6-2fdd-4bc2-a085-99614b6959d0.mov

### PR DESCRIPTION
**Pull request by AyushK-101**
Fixes #4382 
Proper dropdown option selection highlight for dark mode

image

@withshubh I made a few changes, please review

### CODE REVIEW

**Comment by AyushK-101** [comment]
@arpitnath could you please review this

File: frontend/src/_styles/theme.scss [hunk 1]
```diff
@@ -6567,3 +6567,12 @@ tbody {
...
   min-height : 500px;
 } 
+
+.select-search-dark__option.is-selected {
+    background: rgba(45,85,255,1) !important;
+  }
+
+    
+.select-search-dark__option:not(.is-selected):hover {
+      background: rgba(137, 196, 244,1) !important;
```
Comment (arpitnath) [inline frontend/src/_styles/theme.scss:6577]: Can we use 
```suggestion
   background: $primary-rgb-darker !important;
```

**Review by arpitnath — CHANGES_REQUESTED** [review]

**Comment by AyushK-101** [comment]
@arpitnath I've made the required changes

**Comment by arpitnath** [comment]
@AyushK-101 Could you pls resolve the conflicts?

**Review by arpitnath — DISMISSED** [review]
LGTM!

**Comment by arpitnath** [comment]
The issue is not present in the current `develop` branch

**Pull request closed without merging**